runtime/gc_modron_startup/mminit.cpp

```diff
@@ -218,16 +218,26 @@ initializeMutatorModelJava(J9VMThread* vmThread)
 void
 cleanupMutatorModelJava(J9VMThread* vmThread)
 {
-	J9VMDllLoadInfo* loadInfo;
-	J9JavaVM* vm = vmThread->javaVM;
 	MM_EnvironmentBase *env = MM_EnvironmentBase::getEnvironment(vmThread->omrVMThread);
 
 	if (NULL != env) {
+		J9VMDllLoadInfo *loadInfo = NULL;
+		J9JavaVM *vm = vmThread->javaVM;
+
+#if defined(OMR_MIXED_REFERENCES_MODE_STATIC)
+		if (J9JAVAVM_COMPRESS_OBJECT_REFERENCES(vm)) {
+			loadInfo = FIND_DLL_TABLE_ENTRY(J9_GC_DLL_NAME);
+		} else {
+			loadInfo = FIND_DLL_TABLE_ENTRY(J9_GC_FULL_DLL_NAME);
+		}
+#else
+		loadInfo = FIND_DLL_TABLE_ENTRY(J9_GC_DLL_NAME);
```

Comment:
This could be done without the `#else`. The pattern used for most of the rest of the code is:
```
#if static
if (!compressed) {
  info = gc_full
} else
#endif 
{
  info = gc
}
```

Reply:
Is there a preference that the pattern used in jvminit, dllinit, etc. is the same as that of mminit?

This is the current pattern used in jvminit, dllinit, etc:
```
	const char *gcDLLName = gc;
#if static
	if (!compressed) {
		gcDLLName = gc_full;
	}
#endif /*

/* some code in between */

	load(gcDLLName);
```

I can use this pattern for mminit as well, or update mminit with the pattern you've indicated (but will differ from the other files).

Reply:
I thought @dmitripivkine wanted this moved into a helper, and you had agreed. For the simple string assignment, I'm fine with the existing pattern. For the code that calls the macro, I would prefer the new pattern to avoid code duplication (however trivial it may be).